mathesar_ui/src/component-library/button/Button.scss

```diff
@@ -11,6 +11,8 @@
   font-weight: inherit;
   position: relative;
   padding: 0.5em 0.5em;
+  gap: 0.25em;
+  transition: all 0.2s ease-in-out;
```

Comment:
I'm no expect on CSS transitions, but my understanding is that `all` is bad practice and can lead to performance problems. Better to target only the specific properties that you intend to transition.